src/grammar/RegExp.ne

```diff
@@ -0,0 +1,219 @@
+@preprocessor esmodule
+
+Pattern ->
+    Disjunction
+
+Disjunction ->
+    Alternative
+  | Alternative  "|" Disjunction
+
+Alternative ->
+    null
+  | Alternative Term
+
+Term ->
+    Assertion
+  | Atom
+  | Atom Quantifier
+
+Assertion ->
+    "^"
+  | "$"
+  | "\\b"
+  | "\\B"
+  | "(?=" Disjunction ")"
+  | "(?!" Disjunction ")"
+  | "(?<=" Disjunction ")"
+  | "(?<!" Disjunction ")"
+
+Quantifier ->
+    QuantifierPrefix
+  | QuantifierPrefix "?"
+
+QuantifierPrefix ->
+    "*"
+  | "+"
+  | "?"
+  | "{" DecimalDigits "}"
+  | "{" DecimalDigits ",}"
+  | "{" DecimalDigits "," DecimalDigits "}"
+
+Atom ->
+    PatternCharacter
+  | "."
+  | "\\" AtomEscape 
+  | CharacterClass 
+  | "(" GroupSpecifier Disjunction ")"
+  | "(?:" Disjunction ")"
+
+SyntaxCharacter ->
+    [$^\\.*+?()[\]{}|]
+
+PatternCharacter ->
+    [^$^\\.*+?()[\]{}|]
+
+AtomEscape ->
+    DecimalEscape
+  | CharacterClassEscape 
+  | CharacterEscape 
+  | "k" GroupName
+
+CharacterEscape ->
+    ControlEscape
+  | "c" ControlLetter
+  | "0" # [lookahead ∉ DecimalDigit]
+  | HexEscapeSequence
+  | RegExpUnicodeEscapeSequence 
+  | IdentityEscape 
+
+ControlEscape ->
+    [fnrtv]
+
+ControlLetter ->
+    [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ]
+
+GroupSpecifier ->
+    null
+  | "?" GroupName
+
+GroupName ->
+    "<" RegExpIdentifierName ">"
+
+RegExpIdentifierName ->
+    RegExpIdentifierStart 
+  | RegExpIdentifierName RegExpIdentifierPart 
+
+RegExpIdentifierStart ->
+    [a-zA-Z]# UnicodeIDStart
```

Comment:
I have no idea how to encode `UnicodeIDStart` (and `UnicodeIDContinue` on line 93)